auth_manager.go

```diff
@@ -115,11 +114,6 @@ func (b *DefaultSessionManager) ResetQuota(keyName string, session *user.Session
 // UpdateSession updates the session state in the storage engine
 func (b *DefaultSessionManager) UpdateSession(keyName string, session *user.SessionState,
 	resetTTLTo int64, hashed bool) error {
-	if !session.HasChanged() {
-		log.Debug("Session has not changed, not updating")
-		return nil
-	}
-
 	v, _ := json.Marshal(session)
```

Comment:
it is unlikely to get error here but ignoring it could get to corrupted data in storage